src/server/pkg/worker/api_server.go

```diff
@@ -605,13 +619,19 @@ func (a *APIServer) Process(ctx context.Context, req *ProcessRequest) (resp *Pro
 		}
 		return nil, err
 	}
+	logger.Logf("finished uploading output")
 	return &ProcessResponse{
 		Tag: &pfs.Tag{tag},
 	}, nil
 }
 
 // Status returns the status of the current worker.
-func (a *APIServer) Status(ctx context.Context, _ *types.Empty) (*pps.WorkerStatus, error) {
+func (a *APIServer) Status(ctx context.Context, _ *types.Empty) (status *pps.WorkerStatus, retErr error) {
+	logger := log.Logger{}
+	logger.Printf("service: worker, method: Status, response: %v, err %v, duration: %v", status, nil, nil)
+	defer func(start time.Time) {
+		logger.Printf("service: worker, method: Status, response: %v, err %v, duration: %v", status, retErr, time.Since(start))
+	}(time.Now())
```

Comment:
I'm a bit dubious about logging things for read only calls. Especially since you're using an empty `Logger` here so these logs won't be recoverable by any of the methods implemented by `get-logs`.